cmd/controller/app/options/options.go

```diff
@@ -120,6 +126,8 @@ func (s *ControllerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.DefaultACMEIssuerDNS01ProviderName, "default-acme-issuer-dns01-provider-name", defaultACMEIssuerDNS01ProviderName, ""+
 		"Required if --default-acme-issuer-challenge-type is set to dns01. The DNS01 provider to use for ingresses using ACME dns01 "+
 		"validation that do not explicitly state a dns provider.")
+	fs.StringVar(&s.DNS01Nameservers, "dns01-nameservers", defaultDNS01Nameservers, ""+
```

Comment:
`dns01-self-check-nameservers` maybe? Not sure if it's such an important differentiation

Reply:
It'd also be good to provide some examples of this flags usage. As it stands, the user will need to guess that they should specify `8.8.8.8:53` here. I imagine many users will specify `8.8.8.8`.

Let's add a `(e.g. 8.8.8.8:53,8.8.4.4:53)` to the help message, and also perform validation in the options Validate function to ensure that the NS's entered are validated and checked on start-up (we don't need to actually query the NS, just ensure that the values inputted are all sensible, e.g. is a valid IPv4/IPv6 address followed by a port number.